scripts/generic-ci-tests.sh

```diff
@@ -99,7 +99,7 @@ case "$TEST_SUITE" in
         ;;
 
     "lms-unit")
-        PAVER_ARGS="--with-flaky --processes=-1 --cov-args='-p' -v --with-xunitmp"
+        PAVER_ARGS="--with-flaky --processes=-1 --cov-args='-p' --with-xunitmp"
```

Comment:
@jzoldak  FYI on this change.  @cpennington made it so Jenkins would show all test failures at once - rather than incrementally (because of log thresholds).